web/db.py

```diff
@@ -140,6 +140,15 @@ def get_last_temp(vin):
             return None
         return res[0]
 
+    @staticmethod
+    def get_range_timestamp():
+        conn = Database.get_db()
```

Comment:
@jlayec if there is no new position, new charge wont be showed. It can be better to compare last charge and last trip and get the latest date ?

Reply:
hello, 
exact but new position entries are added during charge (last_position.properties.updated_at field is updated) as soon as level is changed so looking at position table is the simplest solution.

Reply:
I choose the complicate solution : 
I changed the code to this : 36f59ed3f5414671b1a26e8de8a1cf7f03553339 .
Can you confirm it works for you?

Reply:
Seems good. Will test it soon. The only drawback is that we will have to implement others lines if we add another table in the future (i m wondering to add a refuel table - i will start a discussion on this topic soon)